src/cmake-tools.ts

```diff
@@ -469,8 +527,12 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
     return this.build(target);
   }
 
-  async showTargetSelector(): Promise<string|null> {
-    const drv = await this._cmakeDriver;
+  async showTargetSelector(): Promise<string | null> {
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return "";
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve('');
```

Reply:
Style guide question